package.json

```diff
@@ -26,46 +27,54 @@
   },
   "homepage": "https://github.com/kognise/water.css",
   "devDependencies": {
+    "@babel/core": "^7.4.5",
+    "@babel/preset-env": "^7.4.5",
+    "@changesets/cli": "^2.7.2",
+    "@ungap/url-search-params": "^0.1.2",
     "autoprefixer": "^9.5.1",
     "browser-sync": "^2.26.3",
     "chalk": "^2.4.2",
+    "clipboard-polyfill": "^2.8.1",
     "cssnano": "^4.1.10",
+    "eslint": "^7.1.0",
+    "eslint-config-standard": "^14.1.1",
+    "eslint-plugin-html": "^6.0.2",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-markdown": "^2.0.0-alpha.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prefer-arrow": "^1.2.1",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.1",
+    "favicon-mode-switcher": "^1.0.4",
     "gulp": "^4.0.0",
+    "gulp-babel": "^8.0.0",
     "gulp-bytediff": "^1.0.0",
     "gulp-filter": "^5.1.0",
     "gulp-flatten": "^0.4.0",
     "gulp-postcss": "^8.0.0",
+    "gulp-posthtml": "^3.0.4",
     "gulp-rename": "^1.4.0",
     "gulp-sizereport": "^1.2.1",
     "gulp-sourcemaps": "^2.6.5",
+    "gulp-terser": "^1.2.0",
+    "htmlnano": "^0.2.3",
+    "husky": "^4.2.5",
     "postcss-color-mod-function": "^3.0.3",
     "postcss-css-variables": "^0.12.0",
     "postcss-import": "^12.0.1",
-    "postcss-inline-svg": "^3.1.1",
-    "standard": "^14.3.1",
-    "stylelint": "^11.0.0",
-    "stylelint-config-standard": "^19.0.0"
+    "postcss-inline-svg": "^4.1.0",
+    "posthtml-include": "^1.2.0",
+    "stylelint": "^13.5.0",
+    "stylelint-config-standard": "^20.0.0",
+    "vue": "^2.6.10"
   },
   "browserslist": {
     "legacy": [
       "defaults AND not android 4.4.3"
-    ],
-    "modern": [
```

Comment:
I had a comment about this on #191 but I'll restart it here instead. Before, we had:
- Prefix a bunch of browsers for the legacy build
- Prefix less browsers  for the modern build

Now we have:
- Prefix a bunch of browsers for the legacy build
- Prefix no browsers for the modern build

It may be worth comparing the modern build with / without prefixes to see if we're losing any significant support. I think the majority of our styles are fine and that we're fine to drop support, but maybe it's worth the consideration?

Reply:
I feel like the main thing we have to worry about is IE11 support - I can't test right now but I can look into browser support for the rules we're using.

Reply:
Well we don't have a modern build anymore (all versions support IE) so I think the current config works fine. But we could remove the "legacy" moniker from our browserslist (and the call to autoprefixer) and just make it the default.

Also we're not autoprefixing the CSS of the docs site right now – not needed for the current style rules, but we got a build pipeline w/ PostCSS for the docs already and have autoprefixer installed, so might as well add it there imo. :)

Reply:
Do you know off the top of your head if we're using any styles that aren't supported in IE11 in the main site? It might be possible to completely remove autoprefixer or write a much more restricted config to reduce our bundle size significantly.

Reply:
Not 100% sure, but I don't think removing autoprefixer is a good idea.

Are we applying styles to `::placeholder`? First thing that came to my mind that (I think) needs prefixing